### ISSUE TITLE
Arrays returned by `WrapCySolverResult` can be deleted while refs to arrays are still alive.

### ISSUE THREAD

**Issue by jrenaud90** [jrenaud90/CyRK#80]
As noted in Issue #78, if arrays from `WrapCySolverResult` that are stored in a local variable are merely references to the underlying memory held by the `WrapCySolverResult` instance. If that instance is destroyed, then the variables are no longer pointing to valid memory.

Fix is to add a wrapper `np.copy` around the `np.asarray` in the following return values of the `WrapCySolverResult` class.

```python
@property
def t(self):
    return np.asarray(self.time_view, dtype=np.float64, order='C')
    
@property
def y(self):
    return np.asarray(self.y_view, dtype=np.float64, order='C').reshape((self.size, self.num_dy)).T
```

**Comment by jrenaud90** [comment]
New tests are showing the issue. working on fix

**Comment by dihm** [comment]
An equivalent alternative fix is to replace `np.asarray` with `np.array`. Not sure there is any significant benefit from one way or the other.

**Comment by jrenaud90** [comment]
Hey @dihm - I have a fix in v0.12.1 which is on pypi now (via pip) and will be on conda forge soon. It is passing my tests but give it a try and see if it works for you! If not please reopen this issue

**Comment by dihm** [comment]
That works like a charm. Thanks @jrenaud90!
